UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -822,14 +823,13 @@ <h3>Additional accessibility information</h3>
 				<dl>
 					<dt>Structure</dt>
 					<dd>
-						<p>For information on any structuring aids that facilitate use of a resource (e.g., annotations,
-							ARIA).</p>
+						<p>For information on any structuring aids that facilitate use of a resource (e.g., ARIA).</p>
 					</dd>
 
 					<dt>Adaptation</dt>
 					<dd>
 						<p>For information on provisions in the content that enable reading in alternative access modes
-							(e.g., closed captions, ruby annotations, sign language, transcript).</p>
+							(e.g., closed captions, ruby-annotations, sign language, transcript).</p>
```

Comment:
No need to hyphenate ruby annotations here.

```suggestion
							(e.g., closed captions, ruby annotations, sign language, transcript).</p>
```